From db5940a6cb8f29fae442aa44aecdafa1ab7d31ad Mon Sep 17 00:00:00 2001
From: Kairui Song <kasong@redhat.com>
Date: Mon, 2 Jul 2018 01:14:53 +0800
Subject: [PATCH] asdasd

---
 rts/Interpreter.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rts/Interpreter.c b/rts/Interpreter.c
index 0e80593..14314f2 100644
--- a/rts/Interpreter.c
+++ b/rts/Interpreter.c
@@ -148,7 +148,7 @@
 STATIC_INLINE StgPtr
 allocate_NONUPD (Capability *cap, int n_words)
 {
-    return allocate(cap, stg_max(sizeofW(StgHeader)+MIN_PAYLOAD_SIZE, n_words));
+    return allocate(cap, stg_max((int)sizeofW(StgHeader)+MIN_PAYLOAD_SIZE, n_words));
 }
 
 int rts_stop_next_breakpoint = 0;
@@ -1832,7 +1832,7 @@ run_BCO:
 
             // Copy the return value back to the TSO stack.  It is at
             // most 2 words large, and resides at arguments[0].
-            memcpy(Sp, ret, sizeof(W_) * stg_min(stk_offset,ret_size));
+            memcpy(Sp, ret, sizeof(W_) * stg_min(stk_offset,(int)ret_size));
 
             goto nextInsn;
         }
-- 
2.16.4

